### ISSUE TITLE
"Experienced a Disruption"

### ISSUE THREAD

**Issue by ertyseidohl** [richardharrington/disrupt-to-bullshit#1]
IMO "Experienced a Disruption" should translate to "Shit the Bed". It would be hilarious.

See: http://www.nytimes.com/2015/04/18/business/dealbook/bloomberg-terminals-outage.html?_r=0

E.G.

`In a statement early in the European afternoon, Bloomberg said: “Significant but not all parts of our network experienced a bullshit today. We have restored service to most customers and are making progress in bringing all parts of the network back online. We apologize to our customers.”`

becomes

`In a statement early in the European afternoon, Bloomberg said: “Significant but not all parts of our network shit the bed today. We have restored service to most customers and are making progress in bringing all parts of the network back online. We apologize to our customers.”`

**Comment by richardharrington** [comment]
Hey Erty! 

I appreciate the sentiment and I'm glad you are still keeping an eye out for fine-tuning this DIY natural language processor.

However, in this particular case I disagree with this change, for two reasons:

1) Because for consistency, I think every transformation should contain the word "bullshit" in it, not just "shit."

2) More broadly, because the example in the text is an instance of the actual, original meaning of the word "disruption." Ideally, the processor would filter out all such uses of the word, and not transform them to different forms of "bullshit." I made some stabs at that early on, and it was too difficult so I abandoned it. I admit that the fact that it actually does get transformed is less than ideal, but I don't think it's an optimal solution to come up with specialized transformations for it, which I think draws away from the main point of the plugin: to transform the word "disrupt" in the Clayton Christensen "disruptive innovation" sense, as popularized by the business and tech press.
